modules/features2d/perf/perf_feature2d.cpp

```diff
@@ -12,6 +12,8 @@ PERF_TEST_P(feature2d, detect, testing::Combine(Feature2DType::all(), TEST_IMAGE
     declare.in(img);
     Mat mask;
     vector<KeyPoint> points;
+    // initialize task scheduler for TBB
+    cv::setNumThreads(cv::getNumberOfCPUs());
```

Comment:
this is useful to get consistent results with instrumentation (first parallel function does not take the initialization penalty). But if you don't like this hack, it can be removed, just the timing will be less stable.